# -- BEGIN LICENSE BLOCK ----------------------------------------------
# -- END LICENSE BLOCK ------------------------------------------------

TrackedCar[] cars
TrackedPerson[] persons
TrackedObject[] obstacles
TrafficSign[] trafficSigns
int16 follow_car_id
